README.rst

```diff
@@ -0,0 +1,68 @@
+#################
+sample-avro-alert
+#################
+
+This package provides information about, and utilties for working with, LSST alerts in `Apache Avro`_ format.
+In particular, it includes:
+
+- Alert schemata;
+- Examples of alert contents;
+- Utilities code for working with alerts.
+
+Refer to `DMTN-093`_ for more information on the LSST alert format.
+
+.. _Apache Avro: https://avro.apache.org
+.. _DMTN-093: https://dmtn-093.lsst.io
+
+Schemata
+========
+
+Alert schemata are located in the ``schema`` directory.
+
+Schemata are filed according to their version number, following a ``MAJOR.MINOR`` scheme.
+We maintain ``FORWARD_TRANSITIVE`` compatibility within a major version, per the `Confluent compatibility model`_.
+The latest version of the schema may always be found at ``schema/latest``.
+
+.. _Confluent compatibility model: https://docs.confluent.io/current/schema-registry/docs/avro.html#forward-compatibility
+
+Example Alert Contents
+======================
+
+Example alert contents are provided in the ``sample_data`` directories included with the corresponding schema.
+
+Utility Code
+============
+
+All code is written in Python, and uses the `fastavro`_ library.
```

Comment:
Slightly unhappy with that, because it opens up so many other questions: do folks have permissions to write to the default installation directory, should they actually be using Conda, ...

I'll add a note that “these can both be installed using the standard methods for Python libraries (pip, Conda, etc)”.